src/temporal/t.rast.aggregate.seasons/testsuite/test_seasons.py

```diff
@@ -0,0 +1,117 @@
+"""Test for t.rast.seasons
+(C) 2023 by the GRASS Development Team
+This program is free software under the GNU General Public
+License (>=v2). Read the file COPYING that comes with GRASS
+for details.
+@author Luca Delucchi
+"""
+
+import grass.temporal as tgis
+from grass.gunittest.case import TestCase
+from grass.gunittest.main import test
+
+
+class TestClimatologies(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        """Initiate the temporal GIS and set the region"""
+        tgis.init(True)  # Raise on error instead of exit(1)
```

Comment:
Using a named (keyword) argument may remove the need for a comment:

```suggestion
        tgis.init(raise_fatal_error=True)
```